src/index.html

```diff
@@ -15,8 +15,49 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="logo-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="navigation">
+        <ul class="nav-links">
+          <li>
+            <a
+              href="#"
+              class="is-active"
```

Comment:
Fix classes everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
```